src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -69,21 +69,25 @@
         programGroup = Metrics.class
 )
 public class CollectAlignmentSummaryMetrics extends SinglePassSamProgram {
-    static final String USAGE_SUMMARY = "Produces a file containing summary alignment metrics from a SAM or BAM.";
-    static final String USAGE_DETAILS = "<br />" +
+    static final String USAGE_SUMMARY = "Produces a file containing summary alignment metrics from a SAM or BAM file";
+    static final String USAGE_DETAILS = "Using read outputs from HTS technologies, this tool provides detailed metrics " +
```

Comment:
Is HTS a well-known acronym?  (Perhaps I am under-caffeinated, but it took me a while to figure out what this meant.  Maybe spell it out.